File: content/polympiads.2/5.controlleur-optimal/.codes/empty.java
import java.util.Scanner;

class Main { 
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
		int nbArrets = sc.nextInt();

		int[] parents = new int[nbArrets];
		for (int i = 0; i < nbArrets; i ++) {
			parents[i] = sc.nextInt();
		}
		
		int nbVoyageurs = sc.nextInt();
	
		int[] arretDepart = new int[nbVoyageurs];
		int[] arretFin = new int[nbVoyageurs];
		for (int i = 0; i < nbVoyageurs; i ++) {
			arretDepart[i] = sc.nextInt();
			arretFin[i] = sc.nextInt();
		}
		
		// Affichez pour tous les arrêts le nombre de voyageurs qui y passent
	}
}

File: content/polympiads.2/5.controlleur-optimal/.codes/empty.cpp
#include <iostream>
#include <vector>

using namespace std;

int main () {
    int nbArrets;
    cin >> nbArrets;

    vector<int> parents(nbArrets);
    for (int i = 0; i < nbArrets; i ++) {
        cin >> parents[i];
    }

    int nbVoyageurs;
    cin >> nbVoyageurs;

    vector<int> arretDepart (nbVoyageurs);
    vector<int> arretFin (nbVoyageurs);
    for (int i = 0; i < nbVoyageurs; i ++) {
        cin >> arretDepart[i] >> arretFin[i];
    }
    
    // Affichez pour tous les arrêts le nombre de voyageurs qui y passent
}
